contrib/endpoints/src/api_manager/check_security_rules.cc

```diff
@@ -17,14 +17,12 @@
 #include <iostream>
 #include <sstream>
 #include "contrib/endpoints/src/api_manager/auth/lib/json_util.h"
-#include "contrib/endpoints/src/api_manager/proto/security_rules.pb.h"
+#include "contrib/endpoints/src/api_manager/firebase_rules/firebase_request.h"
```

Comment:
I initially started off with authz but then decided to firebase_rules since this is very explicit as to what is being used. 
1) I don't think auth is the right place for this since authentication and authorization are two different things.
2) Here are few ways things can go: 
a) If we continue using firebase rules for authorization, then this is directory is still good. 
b) If we decide to support another form of authz along with firebase rules, then I think we can create a directory called authz and move the firebase_rules under that directory and add another directory that appropriately names the authorization mechanism we will support.
c) We decide to discontinue using firebase rules, then we delete this directory. 

I think keeping this as firebase_rules makes a lot of sense since that is what we are doing.